src/unix/osd.c

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2015 Los Alamos Nat. Security, LLC. All rights reserved.
```

Comment:
Missing Intel copyright addition here? (and possibly on some of the other files you modified)